v2/azure-arm.yaml

```diff
@@ -3,11 +3,22 @@ schemaUrlRewrite:
   from: 'https://schema.management.azure.com/schemas/'
   to: 'specs/azure-resource-manager-schemas/schemas/'
 destinationGoModuleFile: go.mod
-# These two paths are relative to the module path above.
+
+# typesOutputPath specifies the output folder name, relative to the directory containing the mod file above.
 typesOutputPath: api
+
+# typeRegistrationOutputFile specifies the output file name for registration code, relative to the directory 
+# containing the mod file above.
 typeRegistrationOutputFile: internal/controllers/controller_resources_gen.go
+
+# samplesUrl is the base URL used for linking to samples when we generate a list of supported resources
 samplesUrl: https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples
+
 pipeline: azure
+
+#
+# These filters are used early in the processing pipeline to simplify 
```

Comment:
to simplify what?

Reply:
_Everything_. 😁 

I'd made changes but not saved them. Will update.